README.md

```diff
@@ -1,8 +1,12 @@
+<p align="center">
+<img src="./docs/images/cover.png" height="220" />
+</p>
+
 # Widget Test Composer
 
 ## Description
 
-Utility that simplifies widget and golden testing.
+Widget Test Composer is a utility package designed to simplify widget and golden testing processes for Flutter applications. Developed by [Surf :surfer:](https://surf.dev/flutter/) Flutter team :cow2:, it offers comprehensive features to facilitate efficient testing workflows.
```

Comment:
Nowhere does it say that it simplifies the configuration for golden_toolkit

Reply:
Add